example/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "typedoc-example",
+  "version": "0.0.0",
+  "private": "true",
```

Comment:
Shouldn't this be a boolean?

Reply:
Good point! Sorry about the CI failures BTW. I was just curious if they would run on a draft PR.

Reply:
Not a problem at all :) Doesn't show up on master, so the status badge is still green